tests/test_proxy_functional.py

```diff
@@ -113,6 +117,10 @@ def _pretend_asyncio_supports_tls_in_tls(
     )
 
 
+@pytest.mark.xfail(
+    PY_310 and platform.system() != "Darwin",
+    reason="we need to fix the secure proxy fixture in 3.10",
```

Comment:
Let's be more precise here:
```suggestion
    raises=ConnectionAbortedError,
    reason="we need to fix the secure proxy fixture in 3.10",
```